projects/packages/forms/src/blocks/contact-form/child-blocks.js

```diff
@@ -560,6 +446,9 @@ export const childBlocks = [
 				src: <Icon icon={ globe } />,
 			},
 			edit: editField( 'url' ),
+			save() {
+				return <InnerBlocks.Content />;
```

Comment:
I think it's better to use the `useInnerBlocksProps` hook and not have the extra wrappers.